af/firefox-ios.xliff

```diff
@@ -222,24 +229,19 @@
   </file>
   <file original="Client/BookmarkPanel.strings" source-language="en" datatype="plaintext" target-language="af">
     <header>
-      <tool tool-id="com.apple.dt.xcode" tool-name="Xcode" tool-version="8.3.3" build-num="8E3004b"/>
+      <tool tool-id="com.apple.dt.xcode" tool-name="Xcode" tool-version="9.0" build-num="9M202q"/>
     </header>
     <body>
       <trans-unit id="Delete">
         <source>Delete</source>
         <target>Verwyder</target>
         <note>Action button for deleting bookmarks in the bookmarks panel.</note>
       </trans-unit>
-      <trans-unit id="Desktop Bookmarks">
```

Comment:
See previous comment. Why was this string moved from `Client/BookmarkPanel.strings` to `BookmarkPanel.strings` ... ohhh look at that, Xcode decided to change the filename ...